bin/install.sh

```diff
@@ -204,7 +210,7 @@ install_init_vars() {
     : ${install_version_fedora:=36}
     : ${install_version_python:=3.9.15}
     : ${install_version_python_venv:=py${install_version_python%%.*}}
-    : ${install_version_centos:=7}
```

Comment:
We should have a staged release here. It should work with centos 7 for now and testers should set install_version_centos=9 in their environments. Once alma 9 is in production, we update this default.